integration-cli/docker_cli_build_dockerignore_test.go

```diff
@@ -0,0 +1,254 @@
+package main
```

Comment:
I've thought about it more... and to make it easier to focus just on what's "new", please move all of this back into docker_cli_build_test.go for now.  After this PR we can then decide how we want to split this file up.